pkg/util/cephconf.go

```diff
@@ -48,3 +53,31 @@ func WriteCephConfig() error {
 
 	return ioutil.WriteFile(CephConfigPath, cephConfig, 0640)
 }
+
+/*
+if any ceph commands fails it will log below error message
+
+2019-07-22 08:31:31.540 7f39ff02a700 -1 auth: unable to find a keyring on
+/etc/ceph/ceph.client.admin.keyring,/etc/ceph/ceph.keyring,/etc/ceph/keyring,
+/etc/ceph/keyring.bin,: (2) No such file or directory
+*/
+// CreateKeyRingFiles creates the keyring files to fix above error message logging
```

Comment:
- Call createCephConfigRoot() (so that the parent directories exist)
- Create each file with a mode that is **only** readable by the user creating it and no other, to prevent any security issues in the future (although in the container environment this maybe far less a concern, as a practice would be useful to follow)

Reply:
> * Call createCephConfigRoot() (so that the parent directories exist)
> * Create each file with a mode that is **only** readable by the user creating it and no other, to prevent any security issues in the future (although in the container environment this maybe far less a concern, as a practice would be useful to follow)


as these files are created inside container not available outside. so this won't be an issue